target/consensource-api
# Fix coveralls report error

Signed-off-by: AdeebAhmed <adeeb.ahmed@target.com>

## Proposed change/fix

Tarpaulin was successfully generating the test coverage report. The issue was the report was not getting set to the coveralls.io service. This PR maps the proper ports to the host container and uses the `$COVERALLS_TOKEN` to send the report the appropriate repo (target/consensource-api).

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

See build:
https://travis-ci.org/target/consensource-api/builds/632014840?utm_medium=notification&utm_source=github_status

Badge updated with the correct percentage: [![Coverage Status](https://coveralls.io/repos/github/target/consensource-api/badge.svg?branch=aahmed-coveralls-fix)](https://coveralls.io/github/target/consensource-api?branch=aahmed-coveralls-fix)
